mne_bids/inspect.py

```diff
@@ -1,12 +1,10 @@
 from pathlib import Path
-import matplotlib.pyplot as plt
-from matplotlib.widgets import Button
 
 from mne_bids import read_raw_bids, mark_bad_channels
 from mne_bids.read import _from_tsv
 
 
-def inspect(bids_path, verbose=None):
+def inspect_bids(bids_path, block=True, verbose=None):
```

Comment:
shall we call this inspect raw? unless you foresee inspection of more types of files?

Reply:
Yeah I thought we could also add an MRI viewer and such… WDYT?

Reply:
if you want to go generic maybe you can do:

inspect_dataset(bids_path, ..., datatype=('meg', 'eeg'))

so we can decide how to inspect differently datatypes

here it would inspect all the files ending with _eeg or _meg

wdyt?

Reply:
How would we go about multiple matches? Would we just iterate over all of them?

Reply:
yes first do meg then eeg etc.

Reply:
Currently the doctoring says:

```
    bids_path : BIDSPath
        A :class:`mne_bids.BIDSPath` containing enough information to uniquely
        identify the raw data file. If this ``BIDSPath`` is accepted by
        :func:`mne_bids.read_raw_bids`, it will work here.
```
I think this is fair… WDYT?

Reply:
Seems I missed your previous response, but I also thought about this again and now I agree, we should iterate over all matches and not force the user to specify a BIDSPath for each file individually. I will implement a solution.